src/App.js

```diff
@@ -1,12 +1,87 @@
-import React from 'react';
-import './App.css';
+
+import React, { useEffect, useState } from 'react';
+import axios from 'axios';
+import PokemonCard from './components/PokemonCard/PokemonCard';
 
 function App() {
-  return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
-  );
+    const [pokemonList, setPokemonList] = useState([]);
+    const [nextLink, setNextLink] = useState('');
+    const [previousLink, setPreviousLink] = useState('');
+    const [loading, setLoading] = useState(false)
+    const [error, setError] = useState(false)
+
+    useEffect(() => {
+        async function getPokemon() {
+            setLoading(true)
+            try {
+                const response = await axios.get(`https://pokeapi.co/api/v2/pokemon/?limit=20`);
+                console.log(response.data);
+                setPokemonList(response.data.results);
+                setNextLink(response.data.next);
+                setPreviousLink(response.data.previous);
+            } catch (e) {
+                console.error(e);
+                setError(true)
+            }
+            setLoading(false)
+
+        }
+
+
+        void getPokemon();
+    }, []);
+
+    const handleNext = async () => {
+        try {
+            const response = await axios.get(nextLink);
+            console.log(response.data);
+            setPokemonList(response.data.results);
+            setNextLink(response.data.next);
+            setPreviousLink(response.data.previous);
+        } catch (e) {
+            console.error(e);
+            setError(true)
+        }
+    };
+
+    const handlePrevious = async () => {
+        try {
+            const response = await axios.get(previousLink);
+            // console.log(response.data);
+            setPokemonList(response.data.results);
```

Comment:
En dan moet dit natuurlijk
```suggestion
            setData(response.data);
```
worden